ansible/bambichecker.yml

```diff
@@ -11,13 +11,13 @@
       - wireguard
       - role: filebeat
         vars:
-          elk: 192.168.1.0
+          elk: {{ elk_adress }}
```

Comment:
Where is that variable configured? If this is done in the config file please update the `README.md` accordingly

Reply:
already done